src/util/jira-utils.ts

```diff
@@ -114,3 +127,73 @@ export const hasJiraIssueKey = (str: string): boolean => !isEmpty(jiraIssueKeyPa
 export const isGitHubCloudApp = async (gitHubAppId: number | undefined): Promise<boolean> => {
 	return !(gitHubAppId && await GitHubServerApp.getForGitHubServerAppId(gitHubAppId));
 };
+
+const deleteSecurityWorkspaceLinkAndVulns = async (
+	installation: Installation,
+	subscription: Subscription,
+	logger: Logger,
+) => {
+
+	try {
+		logger.info("Fetching info about GitHub installation");
+
+		const jiraClient = await JiraClient.getNewClient(installation, logger);
+		await Promise.allSettled([
+			jiraClient.deleteWorkspace(subscription.id),
+			jiraClient.deleteVulnerabilities(subscription.id)
+		]);
+	} catch (err: unknown) {
+		logger.warn({ err }, "Failed to delete security workspace or vulnerabilities from Jira");
+	}
+};
+
+export const removeSubscription = async (
+	installation: Installation,
+	ghInstallationId: number,
+	gitHubAppId: number | undefined,
+	req: Request,
```

Comment:
Please try not to take `req` and `res`.
A rule of thumb will be `require as less as you can` when writing function args.
For `req`, I guess we just need a `log`, and for `res`, I guess we don't need anything, the function can just return a `result: succes or not` (And don't return `202`, as this is HTTP concept, should not be related to non-http service/utils code

Reply:
- Removed the `res` and updated `req` to `req.log`
- Updated this method to return boolean value, which is then used to return the response.